File: 01-KotlinBasic/src/test/java/net/franckbenault/MainTest.java
package net.franckbenault;

import org.junit.Test;
import static org.junit.Assert.*;

public class MainTest {

    @Test
    public void testMax() {
        assertEquals(MainKt.max(5, 9), 9);
    }

    @Test
    public void testMax2() {
        assertEquals(MainKt.max2(56, 9), 56);
    }
}

File: 01-KotlinBasic/src/test/kotlin/net/franckbenault/maintest.kt
package net.franckbenault

import org.junit.Test;
import kotlin.test.assertEquals

class MainKotlinTest {

    @Test
    fun testMax() {
        assertEquals(max(5, 9), 9)
    }

    @Test
    fun testMax2() {
        assertEquals(max2(56, 9), 56)
    }
}